Makefile

```diff
@@ -316,19 +316,58 @@ stat: 		## display processes with PID(s) of gunicorn instance(s) running the app
 ## --- Docker targets --- ##
 
 .PHONY: docker-info
-docker-info:	## tag version of docker image for build/push
-	@echo "Docker image will be built, tagged and pushed as:"
-	@echo "$(DOCKER_TAG)"
```

Comment:
Since `DOCKER_TAG` and `LATEST_TAG` are not used anymore (tags made with multiple suffixes), the variables at the top should be removed. 
Image details `-base`, `-webservice` and `-worker` variants should be listed in the `info` target instead of `DOCKER_TAG`.

Reply:
A cleanup is indeed required in the makefile.